drafts/2025-03-28-burnout-and-recovery.md

```diff
@@ -0,0 +1,40 @@
+
+I had known for a while that I should look for a new job, but was paralyzed by burnout that had spiked from festering to overwhelming. In 2021, when Eric reached out, I was at a low point. My mental and physical health were in the dumps. I had been amazed that such a job existed when Eric joined Apsis years before. Given this incredible opportunity, I felt a lot of internal pressure to impress the team. Thankfully, I was able to push through burnout and earn a spot on the team.
+
+At Apsis, the general burnout triggers experienced so commonly in soul-crushing corporate jobs are the guidelines for how *not* to treat employees. While not infallible, Apsis has been an excellent place for me to recover and thrive. 
+
+1. Workload—This is a priority for Apsis. See [work-sustainably] and [don-t-be-a-hero] to hear from our founders on this.
+1. Perceived Autonomy—Apsis is, in every way it can be, about giving the individual control back. Whether that control being in your schedule or about what work you take on. It's fully remote, so you can be a digital nomad. We make every effort to give everyone a chance to choose the projects they work on. 
+1. Community—We are a close-knit team and all end up working with each other on different projects[^4]. We only judge each other when someone overworks (and then only lightly since we're only human). We've supported each other through a variety of life events necessitating extended time off with full pay[^5]. Each Monday during our planning meeting, every team member is given the chance to voice any concerns about how the last week went.
+1. Reward—Most jobs will dangle money and prestige as the reward for your efforts but, Apsis believes in fair pay for more time with your family or to chase other ambitions.
+1. Perceived Fairness—In a world where there are endless examples of companies exploiting workers, it can seed a bias to see every decision you don’t like as unfair. In the matter of burnout, it’s the perception of unfairness that’s the true contributing factor. Employees at Apsis know that the company cares about treating us collectively as well as possible, as such fairness isn’t an issue.
+1. Shared Values—We’ve proven ourselves willing to turn down work that we don’t feel matches our shared values. More importantly, each of the other points on this list is part of our shared ethos to take care of each other.
+
+The main risk of burnout in this environment is self-imposed internal pressures[^6].
+
+I let myself spiral way too far into burnout before taking corrective action. Between support from my now-wife, the Apsis team, and the freedom to once again leave the house, I feel so much better. I gained time and motivation to focus on improvement of my internal and external experience. I now maintain a consistent workout schedule; I contribute more to the household instead of pushing off low-priority chores; it’s only on rare occasions that I commiserate with Garfield on Mondays.
```

Comment:
I realize this post is about burnout but the word feels like it appears a little too often to me 🤷 That's just a gut-check thing but there are three here within a couple sentences, and a similar situation above. I think you can safely elide several of them without other real changes (like here, "spiral way too far" stands fine on its own).